fullstack-cert/html-projects/lab-movie-review-page/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Movie Review</title>
+</head>
+<body>
+        <section class="review">
+            <h1>Movie Review: Rise Beyond</h1>
+            <img src="rise-beyond-2.webp" alt="Rise Beyond Movie Cover">
```

Comment:
this image will need to be added to our cdn repo
https://github.com/freeCodeCamp/cdn

then once that is merged in, the `src` URL will need to be replaced

Reply:
have opened a PR for that: https://github.com/freeCodeCamp/cdn/pull/289